server/routes/userRoute.js

```diff
@@ -0,0 +1,8 @@
+import { Router } from "express";
+import { userLogin } from "../controllers/userController";
+
+const routeUser = Router();
+
```

Comment:
Why not name this `userRouter` to make it consistent with what has already been done?
We already have `authRouter, socialAuthRouter and mainAppRouter`

Reply:
I agree userRoutes 😄